smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HTTPResponseBindingErrorInitGenerator.kt

```diff
@@ -0,0 +1,85 @@
+package software.amazon.smithy.swift.codegen.integration.httpResponse
+
+import software.amazon.smithy.codegen.core.Symbol
+import software.amazon.smithy.model.knowledge.HttpBinding
+import software.amazon.smithy.model.shapes.Shape
+import software.amazon.smithy.model.shapes.StructureShape
+import software.amazon.smithy.swift.codegen.SwiftDependency
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.declareSection
+import software.amazon.smithy.swift.codegen.integration.HTTPProtocolCustomizable
+import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
+import software.amazon.smithy.swift.codegen.integration.HttpBindingResolver
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.SectionId
+import software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits.HTTPResponseTraitPayload
+import software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits.HTTPResponseTraitQueryParams
+import software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits.HTTPResponseTraitResponseCode
+import software.amazon.smithy.swift.codegen.integration.serde.readwrite.addImports
+import software.amazon.smithy.swift.codegen.integration.serde.readwrite.responseWireProtocol
+
+class HTTPResponseBindingErrorInitGenerator(
```

Comment:
Writes bindings for modeled error structs.

Used to be for XML, now used by by all protocols.